src/pages/cli/graphql/data-modeling.mdx

```diff
@@ -132,6 +132,37 @@ query MyQuery {
 }
 ```
 
+Beginning in Amplify CLI version 8.5.0, there is a feature flag named `enableautoindexquerynames`. If enabled, this flag allows configuration of secondary indexes without any explicit naming.
+
+For more information on feature flags, refer to the [feature flags reference](/cli/reference/feature-flags/).
```

Comment:
Given that this is the base case moving forward, we should start the docs with this rather than end on it. Specifying the index name explicitly should be the "advanced" case and follow after. (See similar paradigm with relationships setup.)

Reply:
Will do!